frontend/app/adapters/user-favourite-team.js

```diff
@@ -0,0 +1,21 @@
+import ApplicationAdapter from "./application";
+
+export default class UserFavouriteTeamAdapter extends ApplicationAdapter {
+  namespace = "api/teams";
+
+  urlForCreateRecord(modelName, snapshot) {
+    return `/${this.namespace}/${snapshot.belongsTo("team", {
+      id: true
```

Comment:
Identifier as boolean?

Reply:
Right. This is kind of confusing, since we only have access to the `snapshot` we have to access the team using `belongsTo()` and because we only want the id we use this Object to specify so.
https://api.emberjs.com/ember-data/3.19/classes/Snapshot/methods/belongsTo?anchor=belongsTo

Reply:
instead of creating an adapter here, it might be just ok to use fetch !
a model is not really required, is it?
it's more an action ... favorite this team !